### PR TITLE
Several types for matching allowed

### PR DESCRIPTION
**Pull request by akloeckner**
**Description:**
Just so the docs match the change referenced below...

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17922

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/binary_sensor.pilight.markdown [hunk 1]
```diff
@@ -49,11 +49,11 @@ name:
 payload_on:
   description: "Variable `on` value. The component will recognize this as logical '1'."
   required: false
-  type: string
+  type: string, float, or int
```
Comment (frenck) [inline source/_components/binary_sensor.pilight.markdown]:
```suggestion
  type: [string, float, integer]
```

File: source/_components/binary_sensor.pilight.markdown [hunk 2]
```diff
@@ -49,11 +49,11 @@ name:
...
 payload_off:
   description: "Variable `off` value. The component will recognize this as logical '0'."
   required: false
-  type: string
+  type: string, float, or int
```
Comment (frenck) [inline source/_components/binary_sensor.pilight.markdown]:
```suggestion
  type: [string, float, integer]
```

**Review by frenck — DISMISSED** [review]
Thanks for the PR, @akloeckner!
I've reviewed it and left two small comments.
Could you take a look? 👍

**Comment by akloeckner** [comment]
Nice feature @frenck. I didn't know the "suggested change" until now...

**Review by fabaff — APPROVED** [review]
Thanks :bird:

**Pull request merged**